md/ubuntu-openjdk-tomcat-postgresql.md

```diff
@@ -70,19 +70,19 @@ Prepared transactions is disabled by default in PostgreSQL. You need to enable i
 
 ## Application Server
 
-### Install Tomcat 7
+### Install Tomcat 8.5
 
-To install the Apache Tomcat 7 you need to install the `tomcat7` package:
+To install the Apache Tomcat 8.5 you need to install the `tomcat8` package:
 
-* Run the following command line: `sudo aptitude install tomcat7`
+* Run the following command line: `sudo aptitude install tomcat8`
```

Comment:
This will install Tomcat 8.0 (version provided in Ubuntu Xenial 16.04 LTS) and not Tomcat 8.5. Did you test that the version of Tomcat files we provide in deploy bundle didn't changed between Tomcat 8.0 and Tomcat 8.5?